### DISCUSSION TITLE
How to make terminal transparent

### DISCUSSION THREAD

**Discussion by jim4067** [Eugeny/tabby#7986] [Q&A] [answered]
How can I make the terminal transparent?

I have tried using CSS `background-colour: transparent`, but it did not yield any results.

**Comment by jim4067** [comment, marked as answer]
Found it. 

Had to go to the `window tab` and active on `acrylic background` option.

**Comment by lawweiliang** [comment]
@jim4067 
Please close the discussion if the question was answered. 
It would be helpful to make the discussion section clean. 

Click `mark as answered` and `close the discussion`. 😄 

![image](https://user-images.githubusercontent.com/34936929/228098333-222ac3d4-4c39-4c26-a2a8-52906647e00e.png)
